cppcache/integration-test/testThinClientSecurityCQAuthorizationMU.cpp

```diff
@@ -138,13 +138,15 @@ class MyCqListener : public CqListener {
 };
 
 std::string getXmlPath() {
-  char xmlPath[1000] = {'\0'};
-  const char *path = std::getenv("TESTSRC");
-  ASSERT(path != nullptr,
+  std::string path = std::string(std::getenv("TESTSRC"));
```

Comment:
Same comment as above regarding code safety for std::getenv

Reply:
N/A. Test removed.